packages/core/admin/admin/src/pages/App/index.js

```diff
@@ -128,7 +128,7 @@ function App() {
 
           try {
             const event = 'didInitializeAdministration';
-            await post(
+            post(
               'https://analytics.strapi.io/api/v2/track',
               {
```

Comment:
The error handling won't work properly anymore, because if you don't await the Promise, the surrounding try/ catch block won't catch it.

If you want to move this forward we/d have to remove the surrounding try/ catch and add a `.catch()` clause to the post handlers.